### DISCUSSION TITLE
[v5] table data streaming... How?

### DISCUSSION THREAD

**Discussion by rbartholomay** [adonisjs/core#2025] [Help]
Hi there,

i must import a large table via adonisjs using the mssql driver. When i look into their docu it´s recommend to use streaming for this:

[https://github.com/tediousjs/node-mssql#streaming](https://github.com/tediousjs/node-mssql#streaming) 

Sure i can use the native mssql-code from the docu but how to handle streaming in the adonisjs way?

Bye, René

**Comment by rbartholomay** [comment]
After study the source of adonisjs I think that knexQuery do the magic...

**Reply by thetutlage** [reply]
Yes, knexQuery must be used for now. But I will add support for streaming directly to Lucid as well
